operations/onboarding.md

```diff
@@ -0,0 +1,99 @@
+
+There are additional social events at KCCN that TOC members are invited to as a body or as individuals, attendence at these is purely optional.
+
+#### TOC Offsite
+
+The TOC offsite is a relatively new event, happening at least once per year lasting about 1-2 days plus travel. It is focused time, ideally face-to-face but virtual is also an option, for the TOC to come together and perform strategic planning and operational health development of the TOC, community, TAGs, and projects. Many initiatives the TOC has had success in executing are the result of these offsites and thus highlights their importance to the continued growth and sustainment of cloud native.
+
+### Preparedness and communicating availability
+
+TOC members, in accordance with the Charter Section 6(f)iii, are expected to attend and be prepared in-advance of each meeting so that the conversation may be productive and decisions may be reached in a timely manner.  TOC members are **not full-time employees of the CNCF**, they are individuals elected to serve the community - many with full-time jobs for which their employers support their activites - *for the benefit of cloud native and open source*.  As such, we understand that incidents, pressing issues, and other primary work priorities may occur from time-to-time, particularly as many TOC members hold significant technical leadership roles within their organizations, that may inhibit their ability to make every meeting all the time. Further, we are human with personal lives and responsibilities that may temporarily limit our ability to attend and participate in these meetings, e.g. vacations, personal matters, etc. It is expected that every TOC member *will communicate their absence or limited availability* in advance (when possible), but endevour to be present and prepared for these meetings. 
+
+In addition to these meetings, TOC members are expected to meet synchronously or asynchronously with the TAGs for which they liaison, occassionally attending TAG meetings.
+
+### Inactivity of a TOC member
+
+If, for any reason, a TOC member is continuously absent without notice, see 6(f)iii, their voting privileges are *suspended* until such time as their consecutive attendence at two meetings. Due to the nature of our work, the inability of a TOC member to cast an *informed* vote (as occurs by meeting attendence *and* involvement in regular engagement as detailed above) penalizes the entire TOC's ability to regularly achieve quorum.  If after the 4th absence the TOC is not notified, the TOC Chair with support of CNCF Staff will make an attempt to address the absence with the TOC member.  If the Chair and Staff are unsuccessful, the Chair will bring the issue of absence to TOC vote in accordance with z6(f)ii at the next private meeting and provide notice to the Governing Board of the outcome. 
```

Comment:
I don't think it has come up but should there be a TOC chair back up or TOC vice chair in case the the chair is not able move things along (The TOC chair may be on vacation or they may have some other priorities)  Or that responsibility remains with the CNCF staff?  (This may be a discussion item)

Reply:
This is something OpenSSF has and would be a new concept for the CNCF TOC.  Having been absent from the ecosystem due to unforeseen circumstance in that past, this would definitely provide value if it were to exist. I don't know if the charter needs to identify a vice chair or if the TOC can simply say "in their absence".  @amye ?

Reply:
CNCF staff in the past if the chair is out

Reply:
I would like the TOC to explore the idea of a VC (vice chair). Currently, most of the load is on the TOC chair to lead the TOC, interact with the GB, prep for all of these meetings etc. This was working with the initial workload within the ecosystem, but now we have been expanding substantially, hence might be worthwhile to share the load and have someone to shadow and learn these leadership skills

Reply:
Some o[f these responsibilities](https://github.com/cncf/toc/pull/1234/files#diff-93190491f696d97a11813d3c6229d7c4b24c5c06267c496d320d8832cd3b64aaR13) would be great to share with a TOC VC e.g. TAG-TOC interaction

Reply:
Leaving this comment open as this is a topic of discussion within the TOC.

Reply:
Agreed TOC VC would be good to help offload some work from TOC chair and step in if TOC chair is not avail.